packages/design-tokens/package.json

```diff
@@ -15,18 +15,15 @@
   },
   "scripts": {
     "prebuild": "npm run clean",
-    "build": "npm-run-all build:**",
-    "build:gulp": "gulp",
-    "build:scss": "node scripts/scss.js",
-    "build:format": "prettier ./{dist,docs}/*.{js,json,css,scss} --write --no-config",
+    "build": "gulp && node scripts/scss.js && npm run format",
```

Comment:
These change makes the root project not work when running `npm install` - not totally clear why. Looks like `gulp` isn't listed as a root dependency?

Reply:
`lerna bootstrap` or prebuild _should_ be taking care of it. Its just a bit weird since we also require a `lerna clean` in pretest to remove duplicate versions of react. Let me see if adding gulp and theo to the workspace help